<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kolo sreÄ‡e</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="startGumb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hallOfFameGumb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hall of fame</string>
   </property>
  </widget>
  <widget class="QPushButton" name="izlazGumb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Izlaz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pojmoviGumb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Pojmovi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegistracijaIgracaGumb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Registracija Igraca</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
